polybar-scripts/system-bluetooth-simple/README.md

```diff
@@ -0,0 +1,22 @@
+# Script: system-bluetooth-simple
+
+A small bluetooth icon to activate bluetooth and see the current status
+
+![off](screenshots/off.png) ![on](screenshots/on.png) ![conneted](screenshots/connected.png)
+
```

Comment:
rename your screenshots to 1,2,3 ...
also the name of the image should be the script-name